test/unit/uk/gov/hmrc/tariffclassificationfrontend/views/partials/TextWithLineFeedsViewSpec.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.views.partials
+
+import uk.gov.hmrc.tariffclassificationfrontend.views.ViewMatchers._
+import uk.gov.hmrc.tariffclassificationfrontend.views.ViewSpec
+import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.text_with_line_feeds
+
+class TextWithLineFeedsViewSpec extends ViewSpec {
+
+  "Progressing Disclosure" should {
```

Comment:
Think this should be "Text with Line Feeds"

Reply:
Good spot. I've actually removed this partial and the test now and have used Steve's CSS idea to preserve the linefeeds